django/README.md

```diff
@@ -15,4 +17,26 @@ python manage.py migrate
 # Log into database to inspect the new table
 docker exec -t -i cthub_db_1 psql -U postgres
```

Comment:
this can be done the same way as line 8
docker-compose exec db psql -U postgres

Reply:
Nice one. Looks much cleaner.